terraform/testcases/prometheus_mock/ecs_taskdef.tpl

```diff
@@ -0,0 +1,101 @@
+[
```

Comment:
Can we move all the template files under `templates` folder under `terraform` folder? in this case, we can create `prometheus` folder under `templates` folder for this template. The test case folder only leave `.var` and `otconfig.tpl` for all the testcases

Reply:
👍  will do